docs/code-snippets/outlook-snippets.yaml

```diff
@@ -195,6 +253,10 @@ Office.AppointmentCompose.saveAsync:
         "value":"AAMkADI5...AAA=",
         "status":"succeeded"
     }
+Office.AppointmentCompose.seriesId:
+  - |-
+    var seriesId = Office.context.mailbox.item.seriesId; 
+    var isSeries = (seriesId == null);
```

Comment:
Consider adding an explanation as to why the series time can be null (and maybe why you'd be checking it).

Reply:
+1

Reply:
I think  you meant seriesId instead of seriesTime here. It's explained in the docs above the example but I can add it.